docs/index.md

```diff
@@ -1 +1,95 @@
+---
+layout: index
+---
 
+## Overview
+
+Marquez is an open source **metadata service** for the **collection**, **aggregation**, and **visualization** of a data ecosystem's metadata. It maintains the [provenance](https://en.wikipedia.org/wiki/Provenance#Data_provenance) of how datasets are consumed and produced, provides global visibility into job runtime and frequency of dataset access, centralization of dataset lifecycle management, and much more. Marquez<sup>[1](#1)</sup> was released and open sourced by [The We Company](https://www.we.co).
```

Comment:
This is definitely due to my lack of background, but in this overview it states Marquez is about data provenance, but in the link it links to data lineage which is also mentioned below.

I guess my confusion is that data lineage and data provenance seem like the same thing?

Reply:
> I guess my confusion is that data lineage and data provenance seem like the same thing?

@alexlingww, exactly! I use the terms data lineage and data provenance interchangeable. In other words:

> Data lineage includes the concept of an origin for the data—its original source or provenance—and the movement and change of the data as it passes through systems and is adopted for different uses (the sequence of steps within the data chain through which data has passed).

source: https://www.sciencedirect.com/topics/computer-science/data-lineage